src/SprykerMiddleware/Zed/Process/Business/Translator/TranslatorFunction/ArrayToString.php

```diff
@@ -21,7 +21,7 @@ class ArrayToString extends AbstractTranslatorFunction
     ];
 
     /**
-     * @param string $value
+     * @param mixed $value
```

Comment:
according to code only array is valid/expectation

Reply:
Yep, you are right